### PR TITLE
ui: Update peerings docs links

### PR DESCRIPTION
**Pull request by LevelbossMike**
### Description
Updates the links for peering docs to some actual peering related docs. Adding as a draft until we get the ok if the correct docs are linked.

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
- [Documentation on Peers](https://www.consul.io/docs/connect/cluster-peering)
- [Take the tutorial](https://www.consul.io/docs/connect/cluster-peering/create-manage-peering) <- not sure about this one but there was no other peering related tutorial I was able to find.

### PR Checklist

* [ ] ~~updated test coverage~~
* [ ] ~~external facing docs updated~~
* [x] not a security concern

cc @johncowen (please add the docs label as discussed) @evrowe

### CODE REVIEW

File: ui/packages/consul-peerings/app/templates/dc/peers/index.hbs [hunk 1]
```diff
@@ -126,12 +126,12 @@ as |sort filters items|}}
                 <BlockSlot @name="actions">
                   <li class="docs-link">
                     {{!-- what's the docs for peering?--}}
-                    <a href="https://www.consul.io/docs/agent/kv" rel="noopener noreferrer" target="_blank">
+                    <a href="https://www.consul.io/docs/connect/cluster-peering" rel="noopener noreferrer" target="_blank">
```
Comment (johncowen) [inline ui/packages/consul-peerings/app/templates/dc/peers/index.hbs:129]: Oh just noticed, these need to use either `env 'CONSUL_DOCS_URL'` and `env CONSUL_DOCS_LEARN_URL`, or the `t` helper.

Guessing we don't have an i18n in this file yet? If so probably just go with the env stuff if thats easier. Will find you 2 examples already in the codebase in a sec.

Also, do you know where you copy pasted this from, consul.io shouldn't be mentioned anywhere in the codebase apart from the set the single env var?

File: ui/packages/consul-peerings/app/templates/dc/peers/index.hbs [hunk 2]
```diff
@@ -126,12 +126,12 @@ as |sort filters items|}}
                 <BlockSlot @name="actions">
                   <li class="docs-link">
                     {{!-- what's the docs for peering?--}}
-                    <a href="https://www.consul.io/docs/agent/kv" rel="noopener noreferrer" target="_blank">
+                    <a href="{{env 'CONSUL_DOCS_URL'}}/connect/cluster-peering" rel="noopener noreferrer" target="_blank">
```
Comment (im2nguyen) [inline ui/packages/consul-peerings/app/templates/dc/peers/index.hbs:129]:
```suggestion
                    <a href="https://www.consul.io/docs/connect/cluster-peering" rel="noopener noreferrer" target="_blank">
```

File: ui/packages/consul-peerings/app/templates/dc/peers/index.hbs [hunk 3]
```diff
@@ -126,12 +126,12 @@ as |sort filters items|}}
...
                       Documentation on Peers
                     </a>
                   </li>
                   <li class="learn-link">
-                    <a href="https://learn.hashicorp.com/consul/getting-started/kv" rel="noopener noreferrer" target="_blank">
+                    <a href="{{env "CONSUL_DOCS_URL"}}/connect/cluster-peering/create-manage-peering" rel="noopener noreferrer" target="_blank">
```
Comment (im2nguyen) [inline ui/packages/consul-peerings/app/templates/dc/peers/index.hbs:134]:
```suggestion
                    <a href="https://www.consul.io/docs/connect/cluster-peering/create-manage-peering" rel="noopener noreferrer" target="_blank">
```

Comment (LevelbossMike) [reply ui/packages/consul-peerings/app/templates/dc/peers/index.hbs:134]: Thanks for the suggestion but this is what the env - variable is adding. So no need to change this here.

**Comment by LevelbossMike** [comment]
Discussed this with @trujillo-adam - we'll keep this in draft until docs are ready.

File: ui/packages/consul-peerings/app/templates/dc/peers/index.hbs (hunk 2 above)
Comment (LevelbossMike) [reply ui/packages/consul-peerings/app/templates/dc/peers/index.hbs:129]: see https://github.com/hashicorp/consul/pull/13757#discussion_r930709402

File: ui/packages/consul-peerings/app/templates/dc/peers/index.hbs (hunk 3 above)
Comment (im2nguyen) [reply ui/packages/consul-peerings/app/templates/dc/peers/index.hbs:134]: sounds good, thanks!

**Pull request closed without merging**